DuckDuckGo/WindowsBrowserWaitlistView.swift

```diff
@@ -45,7 +45,10 @@ struct WindowsBrowserWaitlistView: View {
                 Task { await viewModel.perform(action: action) }
             }
         case .waitlistRemoved:
-            FailedAssertionView("Windows waitlist is still active")
+            WaitlistDownloadBrowserContentView(platform: .windows) { action in
+                Task { await viewModel.perform(action: action)
+                }
```

Comment:
While you’re fixing the SwiftLint warning reported elsewhere in this PR, please also put this bracket on the line above :)